src/NuGet.Clients/NuGet.Tools/Resources.resx

```diff
@@ -193,4 +193,13 @@
     <value>These settings are stored in %NuGet.Config% files</value>
     <comment>Do not translate % % or what is between them</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="Text_ConfigurationFiles_Priority" xml:space="preserve">
+    <value>Configuration files are listed in order from highest to lowest priority.</value>
+  </data>
+  <data name="Text_ConfigurationFiles_CommonLink" xml:space="preserve">
+    <value>Common NuGet configurations: [How settings are applied](https://aka.ms/nuget/how-settings-are-applied/)</value>
```

Comment:
I'd suggest leaving a comment for localizers to let them know they should leave the square brackets and the URL alone.

Reply:
Yes, good catch!  I did that last time and unfortunately it was still an issue.  I believe our localization process may not propagate comments anyway, but I want to put it here in case that process gets improved